Week2/muhannad.js

```diff
@@ -0,0 +1,144 @@
+// say hello world in 3 language
+console.log('Hello World'); //English
+console.log('Hola Mundo'); //Spanish
+console.log('Bonjour le monde'); //French
+
+//SyntaxError: Find a solution for this error.
+console.log("I'm awesome");
+
+//1 First, declare your variable x (do not initialize it yet).
+let x;
+//3.2 Add a console.log statement that explains in words what you think the value of x is, like in this example:
+console.log(`the value of my variable x will be undefined because it dose not have any value yet`);
+//3.3 Add a console.log statement that logs the value of x.
+console.log(x);
+//3.4 Now initialize your variable x with an integer.
+x = 11;
+//3.5 Next, add a console.log statement that explains what you think the value of x is.
+console.log('the value of x will be 11');
+//3.6 Add a console.log statement that logs the value of x.
+console.log(x);
+//4. Declare a variable y and assign a string to it.
+let y = 'hello javascript';
+//4.1 Write a console.log statement in which you explain in words what you think the value of the string is.
+console.log('the value of x will be: hello javascript');
+//4.2 Now console.log the variable y.
+console.log(y);
+//4.3 Now assign a new string to the variable y.
+y = 'i am studying javascript';
+//4.4 Just like what you did before write a console.log statement that explains in words what you think will be logged to the console.
+console.log('the value of x will be: i am studying javascript');
+//Now console.log y again.
+console.log(y);
+//How do you round the number 7.25, to the nearest integer (i.e., whole number)?
+//5.1 Declare a variable z and assign the number 7.25 to it.
+let z = 7.25;
```

Comment:
You can use `const` here instead of `let`, can't you? Since you aren't re-assigning `z` or `a` anywhere else in your code 😉